doc/manual/manual.tex

```diff
@@ -6570,6 +6592,47 @@ \subsubsection{Reading in compositional initial composition files generated with
 
 An animation of a model using the jellyfish as initial condition and assigning it a higher viscosity can be found here: \url{https://www.youtube.com/watch?v=YzNTubNG83Q}. 
 
+\subsubsection{Using lazy expression syntax for if-else-statements in function expressions}
+\label{sec:lazy-expression}
+\textit{This section was contributed by Magali Billen}\\
+This cookbook provides an example to illustrate how to use the lazy-expression syntax for multiple, nested, if-else-statements in function expressions in the parameter file. It also shows how to set parameters in the input file so you can quickly check initial conditions (i.e., without waiting for the solver to converge).  For this model we define a simple 2D box, which is 5000 x 1000 km, with free-slip velocity boundary conditions. The material parameters are constant within the box (set using the  ``simple'' material model). The initial thermal structure has two parts divided at $xtr=2200$~km. The temperature in each region is defined using the equation for a half-space cooling model:
+\begin{equation}
+T(x,y) = T_s + (T_m  - T_s) erf{(\frac{y}{2\sqrt{\kappa x/v}})}
+\end{equation}
+where $erf$ is the error function, $T_m$ is the mantle temperature, $T_s$ is the surface temperature, $y$ is depth, $x$ is horizontal distance, $\kappa$ is the thermal diffusivity, and $v$ is the plate velocity. The age of the plate is given by $x/v$. Note that the equation for the half-space cooling model is not defined at $x=0$ (because there is a divide by zero inside the error function): at $x=0$, $T=T_m$.  For $(x \le xtr)$ and $(x>0)$ the age of the plate increases from zero at the boundary according to a fixed plate velocity $v_{sub}=7.927\times10^{-10}$~m/s  ($2.5$~cm/yr). This is the subducting plate. For $x > xtr$, there is a fixed plate age of $age_{op}=9.46\times10^{14}$~s ($30$~my); this is the overriding plate. In order to resolve the temperature structure, we also define some initial refinement of the mesh for the top 150 km of the mesh. Both the mesh refinement and the temperature structure are defined using lazy-expression syntax in functions within the parameter file.
+
+Functions can be used in the parameter file to define initial conditions (e.g., temperature, composition), boundary conditions, and even to set region of refinement. We also often want to use different values of functions in different regions of the model (e.g., for the two plates as described above) and so we need to use if-statements to specify these regions. The function constants and expressions are read in using the muparser. The muparser accepts two different syntax for if-statements (see also Section \ref{sec:muparser-format}).
```

Comment:
'set region of refiment' -> 'set a region of refinement' or 'set regions of refinement'

Reply:
"two different syntax": I don't know what exactly the plural of "syntax" is, but maybe one can avoid this by saying "syntax options" or "syntactic options"?

Reply:
plural of syntax is syntaxes (ugh!)... I've changed it to syntax options